### PR TITLE
adding works

### PR DESCRIPTION
**Pull request by whitehorod**

### CODE REVIEW

File: module1/homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result = "Invalid data";
+
+    if( command == "add" ) {
```
Comment (m-kop) [inline module1/homework/calculate/calculate.hpp:7]: please have a look at the code style guide: https://github.com/lefticus/cppbestpractices/blob/master/03-Style.md 

```suggestion
    if (command == "add") { 
```

File: module1/homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result = "Invalid data";
+
+    if( command == "add" ) { 
+      result = std::to_string(first+second);
+    } else
+        if( command == "subtract" ) {
```
Comment (m-kop) [inline module1/homework/calculate/calculate.hpp:10]: it's a good idea to keep "else if" keywords in the same line
```suggestion
        else if (command == "subtract") { 
```

File: module1/homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+        if( command == "multiply" ) { 
+      result = std::to_string(first*second);
+    } else
+        if( command == "divide" ) { 
+          if(second!=0)
+            result = std::to_string(first/second);
+          else
+            result = "Division by 0";
+    } 
+
```
Comment (m-kop) [inline module1/homework/calculate/calculate.hpp:22]: remove additional empty line

File: module1/homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -2,6 +2,23 @@
...
+    } else
+        if( command == "subtract" ) { 
+      result = std::to_string(first-second);
+    } else
+        if( command == "multiply" ) { 
+      result = std::to_string(first*second);
+    } else
+        if( command == "divide" ) { 
+          if(second!=0)
+            result = std::to_string(first/second);
```
Comment (m-kop) [inline module1/homework/calculate/calculate.hpp:18]: it's always a good practice to add { } for a single statement.

File: module1/homework/calculate/calculate.hpp [hunk 5]
```diff
@@ -2,6 +2,23 @@
...
-    return "";
+    std::string result = "Invalid data";
+
+    if( command == "add" ) { 
+      result = std::to_string(first+second);
+    } else
+        if( command == "subtract" ) { 
+      result = std::to_string(first-second);
+    } else
+        if( command == "multiply" ) {
```
Comment (DamiXioR) [inline module1/homework/calculate/calculate.hpp:13]: remove spaces after opening and before closing round bracket

File: module1/homework/calculate/calculate.hpp [hunk 6]
```diff
@@ -2,6 +2,23 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+    std::string result = "Invalid data";
+
+    if (command == "add") { 
+      result = std::to_string(first+second);
```
Comment (piezol) [inline module1/homework/calculate/calculate.hpp:8]: Please use consistent indentation.
4 spaces are used throughout repository.

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1429], "points": [2, 5] }

**Pull request closed without merging**